UserInputs/userInputsData.py

```diff
@@ -1,10 +1,14 @@
+import os
 from pathlib import Path
 
 
 class UserInputsData:
     application = "Village Health"
     domain = "qa-automation"
-    download_path = Path('~/Downloads').expanduser()
+    if os.environ.get("CI") == "true":
+        download_path = r"/home/runner/work/dimagi-qa/dimagi-qa"
+    else:
+        download_path = str(Path('~/Downloads').expanduser())
```

Comment:
I think things will be easier if you keep these as `Path` objects
```suggestion
    if os.environ.get("CI") == "true":
        download_path = Path("/home/runner/work/dimagi-qa/dimagi-qa")
    else:
        download_path = Path('~/Downloads').expanduser()
```
Then you can do things like `download_path / "filename.txt"` to construct a path to a file in the downloads directory.

It should be rare, but occasionally you will find some function that wants a string path rather than a `Path` object. In hose cases you can use `str(path_obj)` to work around that.